stories/Form.jsx

```diff
@@ -22,6 +22,30 @@ storiesOf('Form', module)
       title: 'Form'
     })
   )
+  .addWithInfo('Checkbox', () => {
+    const formGroupContext = select('Context', Form.Group.contexts);
+    const disabled = boolean('Disabled');
```

Comment:
Can you explicitly set `false` for the default value?

Reply:
And maybe add the `checked` prop to the knobs?

Reply:
I don't think this is possible yet.
With the current version of storybook, it would mean to either check the checkboxes only with knobs, or only by clicking the box. We can't do both unless we update the `checked` prop from the story with `onChange` and some kind of story rerendering function.
[For now](https://github.com/storybooks/storybook/releases/tag/v3.4.0-alpha.0), we can't force a story to rerender.

Personally, I would stick with no knobs and check/uncheck directly on the `Form.Checkbox`, since it's the common use.

Reply:
Ok it makes sense thank you!